### PR TITLE
Posted in #channel with truncation.

### PR DESCRIPTION
**Pull request by xPolar**

### CODE REVIEW

File: apps/bot/src/lib/starboard/webhook.ts [hunk 1]
```diff
@@ -31,7 +31,11 @@ export async function createWebhookMessage(
 		.setStyle(ButtonStyle.Link)
 		.setURL(message.url)
 		.setEmoji({ name: "🔗" })
-		.setLabel("View Original");
+		.setLabel(
+			`Posted in #${channel.name}`.length > 80
```
Comment (ThaTiemsz) [inline apps/bot/src/lib/starboard/webhook.ts]:
```suggestion
			`Posted in #${channel.name.length > 69 ? channel.name.slice(0, 66) + "..." : channel.name}`
		);

 	const attachments = [...message.attachments.values()];
```

**Review by ThaTiemsz — CHANGES_REQUESTED** [review]

File: apps/bot/src/lib/starboard/webhook.ts (hunk 1 above)
Comment (IanMitchell) [reply apps/bot/src/lib/starboard/webhook.ts]: nice

File: apps/bot/src/lib/starboard/webhook.ts [hunk 2]
```diff
@@ -31,7 +31,11 @@ export async function createWebhookMessage(
...
+				? `Posted in #${channel.name}`.slice(0, 77) + "..."
```
Comment (IanMitchell) [inline apps/bot/src/lib/starboard/webhook.ts]:
```suggestion
				? `Posted in #${channel.name}`.slice(0, 77) + "…"
```

File: apps/bot/src/lib/starboard/webhook.ts [hunk 3]
```diff
@@ -31,7 +31,14 @@ export async function createWebhookMessage(
...
 		.setEmoji({ name: "🔗" })
-		.setLabel("View Original");
+		.setLabel(
+			`Posted in #${channel.name.length > 69 ? channel.name.slice(0, 66) + "..." : channel.name}`
+		);
+
+ 	const attachments = [...message.attachments.values()];
+				? `Posted in #${channel.name}`.slice(0, 77) + "…"
+				: `Posted in #${channel.name}`
+		);
```
Comment (IanMitchell) [inline apps/bot/src/lib/starboard/webhook.ts]: lol nice merge Ian top tier skill

```suggestion
		.setLabel(
			`Posted in #${channel.name.length > 68 ? channel.name.slice(0, 67) + "…" : channel.name}`
		);

```

**Pull request merged**